pages/articles.tsx

```diff
@@ -16,6 +18,9 @@ export default function Articles({ tags, posts }) {
   );
   return (
     <Layout title={title} description="A list of every article I've written.">
+      <Head>
+        <link rel="stylesheet" href="https://files.stork-search.net/basic.css" />
```

Comment:
`/flat.css` might be better aligned with your site's design! Up to you!

Reply:
Awesome! Fixed.